### ISSUE TITLE
Z-index in menu

### ISSUE THREAD

**Issue by TesterAbzal** [Mango/slideout#228]
Hello! Nice work man :) But can you give me advise about z-index, i want to change z-index by 3, it means that menu don't shift all content in left or right, it  covers the entire site. Like youtube or some web-site

**Comment by impronunciable** [comment]
I think you can do

```
.slideout-panel {
  z-index: 3;
}
```

**Comment by TesterAbzal** [comment]
Yes, but without "position:absolute" it don't work. If i change position to absolute, animation is dropped
